### DISCUSSION TITLE
Getting blocked by CORS error

### DISCUSSION THREAD

**Discussion by ricotrevisan** [esm-dev/esm.sh#472] [General] [answered]
Hi, I've been building a TipTap Editor with ESM all afternoon. Now -- all of a sudden -- it all broke. This codepen was working this afternoon and now it's not --> https://codepen.io/ricotrevisan/pen/RwBNGdQ

Given my limited programming knowledge, I'm quite sure it was my mistake so I didn't want to raise an issue right away, but ask here first if I need to make some changes.

Here's the error:
<img width="503" alt="CleanShot 2022-12-25 at 23 33 43@2x" src="https://user-images.githubusercontent.com/2459327/209483426-3dc03f90-3cb7-4a1e-baed-a5e83da6c714.png">

**Comment by ricotrevisan** [comment]
Found a fix: make sure to add the version of the package.

So, this works:
```
import { Editor } from "https://esm.sh/@tiptap/core@2.0.0-beta.209";
import StarterKit from "https://esm.sh/@tiptap/starter-kit@2.0.0-beta.209";
```

While this doesn't work. Even though it calls the same package.
```
import { Editor } from "https://esm.sh/@tiptap/core";
import StarterKit from "https://esm.sh/@tiptap/starter-kit";
```

**Reply by jeluard** [reply]
FWIW, still have the same issue. And same workaround still applies.
i.e. no CORS headers unless a version is specified

**Reply by ije** [reply]
can you please post the url? thanks!

**Reply by jeluard** [reply]
https://esm.sh/@lodestar/api@1.14.0 => with CORS
https://esm.sh/@lodestar/api => no CORS

**Reply by ije** [reply]
thanks! but seems it works for me
<img width="497" alt="Screenshot 2024-01-17 at 19 37 31" src="https://github.com/esm-dev/esm.sh/assets/2883484/7d1fe42b-3d7d-4f9a-a557-709d441eddb6">

**Reply by jeluard** [reply]
Looks like `access-control-expose-headers` is missing.

See https://www.diffchecker.com/4eXgfWCo/

**Comment by ije** [comment, marked as answer]
i am updating the CDN, that maybe break the cors, i just re-added the `Access-Control-Allow-Origin` header for redirects can you please try again without versioning? thanks!

**Reply by ricotrevisan** [reply]
Yep, works now!

Did I just happen to stumble onto that problem in the 2-minute window of time that you were updating the CDN?
